src/Diagram/index.js

```diff
@@ -41,8 +41,8 @@ const ColumnOne = () => (
     <Description fullWidth>
       <p>
         Version control machine learning models, data sets and intermediate
-        files. DVC connects them with code and uses S3, Azure, Google Drive,
-        GCP, SSH, Aliyun OSS or to store file contents.
+        files. DVC connects them with code, and uses cloud storage, SSH, NAS,
```

Comment:
it's better for SEO to mention names explicitly (I know it looks not good :) )

Reply:
I'm just not sure where this is rendered. I left full names in features.js though. Is this one more general?

Reply:
this the very main landing page I believe

Reply:
OK full list added in 3f1524bc.